lms/djangoapps/discussion_api/tests/test_views.py

```diff
@@ -866,6 +896,31 @@ def test_pagination(self):
             }
         )
 
+    @ddt.data(
+        (True, "endorsed_comment"),
+        ("true", "endorsed_comment"),
+        ("1", "endorsed_comment"),
+        (False, "non_endorsed_comment"),
+        ("false", "non_endorsed_comment"),
+        ("0", "non_endorsed_comment"),
+    )
```

Comment:
@jcdyer @BenjiLee The NullBooleanField's to_python checks for True, "true", "1", False, "false", "0" for valid Boolean values and defaults None to any other value. (reference https://github.com/django/django/blob/master/django/forms/fields.py#L741)
How do you recommend we handle this? Do we override to_python?